kms++/inc/kms++/plane.h

```diff
@@ -7,11 +7,29 @@ namespace kms
 
 enum class PlaneType
 {
+	none = 0,
 	Overlay = 1 << 0,
```

Comment:
My intention was to indicate that none is really not a  planetype, but something different. But I'll change it if you prefer it.